WORKSPACE

```diff
@@ -87,3 +87,7 @@ tf_workspace()
 # Please add all new dependencies in workspace.bzl.
 load("//third_party:workspace.bzl", "tensorboard_workspace")
 tensorboard_workspace()
+
+# Setup TypeScript toolchain.
```

Comment:
nit: As a phrasal verb, [*set up* is two words.][set-up]

[set-up]: https://en.wiktionary.org/wiki/setup#Verb

Reply:
Done. This was copy-pasted from docs on bazelbuild.github.io (https://bazelbuild.github.io/rules_nodejs/TypeScript.html).

Reply:
Thanks: https://github.com/bazelbuild/rules_nodejs/pull/977